artifactory.py

```diff
@@ -862,6 +883,25 @@ def stat(self, pathobj):
             md5=checksums.get("md5", None),
             is_dir=is_dir,
             children=children,
+            repo=jsn.get("repo", None),
+        )
+
+        return stat
+
+    def download_stats(self, pathobj):
+        jsn = self.get_stat_json(pathobj, key="stats")
+
+        # divide timestamp by 1000 since it is provided in ms
```

Comment:
Why do we need to divide it?

Reply:
Otherwise parser cannot parse date properly

Reply:
Oh, I see